aten/src/ATen/functorch/BatchedTensorImpl.cpp

```diff
@@ -89,9 +114,6 @@ void BatchedTensorImpl::set_size(int64_t dim, int64_t new_size) {
 void BatchedTensorImpl::set_stride(int64_t dim, int64_t new_stride) {
   TORCH_INTERNAL_ASSERT(false, "Can't set_stride for BatchedTensorImpl");
 }
-void BatchedTensorImpl::set_storage_offset(int64_t storage_offset) {
```

Comment:
HACK: This is used by `set_sizes_and_strides` above. 

Should we copy the logic from `TensorImpl` there?

Reply:
Ehh I think this is fine. The question is if a User is able to invoke this code.

How would a user invoke this code? I think only via Tensor.as_strided_. But then they'd be setting more things than just the storage offset (like sizes/strides), and that would error out for them.